src/components/ConversationIcon.vue

```diff
@@ -18,15 +18,20 @@
   - You should have received a copy of the GNU Affero General Public License
   - along with this program. If not, see <http://www.gnu.org/licenses/>.
 -->
-
 <template>
-	<div class="conversation-icon"
+	<div ref="conversation-icon"
+		class="conversation-icon"
+		:style="iconStyle"
 		:class="{'offline': offline}">
 		<div v-if="iconClass"
 			class="avatar icon"
+			:style="iconStyle"
 			:class="iconClass" />
+		<NcAvatar v-else-if="hasAvatar"
```

Comment:
Maybe `hasAvatar` should have a priority here, and if it is falsy, check for `iconClass`?
Or it matters when we're waiting for picture to load?